src/App.tsx

```diff
@@ -74,27 +74,19 @@ const App = () => {
   // Create client
   useEffect(() => {
     const getClient = async () => {
-      try {
-        setClientIsLoading(true)
-        // Get clients
-        const clientResp = await createClient(settings)
-        setClient(clientResp)
-
-        console.log('Clients initialized.')
-
-        setSnackbarMessage({
-          text: `Connected to Alephium's Node "${settings.nodeHost}"!`,
-          type: 'info',
-          duration: 4000
-        })
-        setClientIsLoading(false)
-      } catch (e) {
```

Comment:
How do we handle wrong network settings now that you removed the `catch`?

Reply:
It's the outcome of the whole discussion we had on Slack [here](https://alephium.slack.com/archives/C026MHDJD2N/p1637071135044300) and [here](https://alephium.slack.com/archives/C026MHDJD2N/p1637139773058300) and described in the linked task [here](https://github.com/alephium/alephium-wallet/issues/53#issuecomment-971402782).

Returning selfClique errors could be misleading (it has been, for some users), as we previously considered it as a pseudo-health check. 

- For now, we decided to display an error if an TX cannot be send successfully, indicating a bad configuration or network issue.
- When the user update the settings, we simply print a "Settings saved" message.

(BTW, this shows that brainstorming on Slack is probably not the best solution, as the detailed outcomes are easily lost.)

Reply:
sorry, completely forgot about this. Resolving.